Makefile

```diff
@@ -78,7 +92,10 @@ ifndef DEPLOY_ENV
 	$(error DEPLOY_ENV is undefined)
 endif
 
+clean:
+	rm ~/.ssh/id_*
+
```

Comment:
I find that a bit risky. It still depends where the Makefile is executed. If I am running it on my machine, `make clean` will remove my private SSH key (i.e. ~/.ssh/id_rsa).

Reply:
I agree. I forgot to change the naming. Here d577e65